web/src/js/components/Modal/OptionMaster.jsx

```diff
@@ -1,20 +1,24 @@
+import React, { Component } from 'react'
 import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import classnames from 'classnames'
+import { update as updateOptions } from '../../ducks/options'
 
 PureBooleanOption.PropTypes = {
     value: PropTypes.bool.isRequired,
     onChange: PropTypes.func.isRequired,
 }
 
-function PureBooleanOption({ value, onChange, name, help}) {
+function PureBooleanOption({ value, onChange, ...props}) {
+        let onMouseEnter = () => { props.onMouseEnter() },
+            onMouseLeave = () => { props.onMouseLeave() }
```

Comment:
Minor: Is there a particular reason why you don't just pass `props.onMouseEnter` down?
(We can probably also just use `...props` below?)